src/components/DatePicker/index.ios.js

```diff
@@ -1,163 +1,147 @@
-import React from 'react';
-// eslint-disable-next-line no-restricted-imports
+import React, {useState, useRef, useCallback, useEffect} from 'react';
 import {Button, View, Keyboard} from 'react-native';
 import RNDatePicker from '@react-native-community/datetimepicker';
 import moment from 'moment';
-import _ from 'underscore';
-import compose from '../../libs/compose';
+import isFunction from 'lodash/isFunction';
 import TextInput from '../TextInput';
-import withLocalize, {withLocalizePropTypes} from '../withLocalize';
 import Popover from '../Popover';
 import CONST from '../../CONST';
 import styles from '../../styles/styles';
 import themeColors from '../../styles/themes/default';
 import {propTypes, defaultProps} from './datepickerPropTypes';
-import withKeyboardState, {keyboardStatePropTypes} from '../withKeyboardState';
+import useKeyboardState from '../../hooks/useKeyboardState';
+import useLocalize from '../../hooks/useLocalize';
 
-const datepickerPropTypes = {
-    ...propTypes,
-    ...withLocalizePropTypes,
-    ...keyboardStatePropTypes,
-};
+function DatePicker({value, defaultValue, innerRef, onInputChange, preferredLocale, minDate, maxDate, label, disabled, onBlur, placeholder, containerStyles, errorText}) {
+    const [isPickerVisible, setIsPickerVisible] = useState(false);
+    const [selectedDate, setSelectedDate] = useState(value || defaultValue ? moment(value || defaultValue).toDate() : new Date());
```

Comment:
Could we simplify like this?

```suggestion
    const [selectedDate, setSelectedDate] = useState(moment(value || defaultValue || new Date()).toDate());
```

Reply:
Updated to the simpler version that works same